src/Component/BulkEditor/BulkEditor.tsx

```diff
@@ -41,20 +41,13 @@ import { KindField } from '../Symbolizer/Field/KindField/KindField';
 import { SymbolizerKind, WellKnownName } from 'geostyler-style';
 import WellKnownNameField from '../Symbolizer/Field/WellKnownNameField/WellKnownNameField';
 import ImageField from '../Symbolizer/Field/ImageField/ImageField';
+import { GeoStylerLocale } from '../../locale/locale';
 
-// i18n
-export interface BulkEditorLocale {
-  colorLabel: string;
-  radiusLabel: string;
-  opacityLabel: string;
-  symbolLabel: string;
-  imageFieldLabel: string;
-}
 
 // default props
 interface BulkEditorDefaultProps {
   /** Locale object containing translated text snippets */
-  locale: BulkEditorLocale;
+  locale: GeoStylerLocale['BulkEditor'];
```

Comment:
I'm not sure if i like the removal of the `Gs` prefix (`GsBulkEditor` -> `BulkEditor`). We are extending the antd locales so we might run into naming collisions.

Reply:
Currently there are no conflicts... IMHO we can think about this when we run into conflicts.